src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -13,10 +16,20 @@
 @WebServlet("/user-list")
 public class UserListServlet extends HttpServlet {
 
+  private Datastore datastore;
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException {
-
-    response.getOutputStream().println("this will be my user list");
+    response.setContentType("application/json");
+    Set<String> users = datastore.getUsers();
+    Gson gson = new Gson();
```

Comment:
Just wondering, what is gson?

Reply:
I think it is a google library to parse json into java objects